gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/resources/GenerateResClassTask.kt

```diff
@@ -105,6 +104,9 @@ abstract class GenerateResClassTask : DefaultTask() {
             .map { it.attributes.getNamedItem("name").nodeValue }
         return ids.toSet()
     }
+
+    private fun File.listFilesWithoutDsStore(): List<File> =
+        listFiles()?.filter { it.name != ".DS_Store" }.orEmpty()
```

Comment:
I think we can filter any hidden file, not only .DStore (File.isHidden can be used)